core/lib/external_price_api/src/coingecko_api.rs

```diff
@@ -49,6 +49,7 @@ impl CoinGeckoPriceAPIClient {
         }
     }
 
+    /// returns ETH/BaseToken price of a token by address
```

Comment:
nit: It's base to eth, that's why you've had to flip it to be eth / base down in the file

Reply:
So in terms on numerator denominator it's (here) numerator=eth, denominator=base, and it's flipped below. Please if you have any idea, suggest how to comment this properly, so there is no confusion in the future.

Reply:
Yes, you are right, resolving